wrangler-core/src/main/java/io/cdap/directives/column/Rename.java

```diff
@@ -94,4 +85,12 @@ public void destroy() {
     }
     return rows;
   }
+
+  @Override
+  public Mutation lineage() {
+    return Mutation.builder()
+      .readable("Renamed column '%s' to '%s'", source.value(), target.value())
+      .relation(source, Many.columns(source, target))
```

Comment:
this should be a one-to-one mapping since there is no source col after the rename

Reply:
fixed